packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -2,25 +2,18 @@ import React from 'react';
 import getStatusBadgeConfig from '@deriv/account/src/Configs/get-status-badge-config';
 import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
 import { Text, Icon, Money, StatusBadge } from '@deriv/components';
-import {
-    getCFDAccountDisplay,
-    getCFDAccountKey,
-    getCFDPlatformLabel,
-    getPlatformSettings,
-    getUrlBase,
-    MT5_ACCOUNT_STATUS,
-} from '@deriv/shared';
+import { getCFDAccountDisplay, getCFDAccountKey, MT5_ACCOUNT_STATUS } from '@deriv/shared';
```

Comment:
`getCFDAccountDisplay` seems like a very mysterious name

Reply:
Changing this name would require updating 13 files where `getCFDAccountDisplay` is used... 🤔 I would say let's not
![image](https://github.com/binary-com/deriv-app/assets/108507236/5a134f7a-9305-4133-8356-de3e25295a51)

Reply:
Changing this name would require updating 13 files where `getCFDAccountDisplay` is used... 🤔 would say let's not 😬
![image](https://github.com/binary-com/deriv-app/assets/108507236/f535a0b1-3807-42b7-9add-454dae50c0f7)